test/jit/test_symbolic_shape_analysis.py

```diff
@@ -573,3 +573,14 @@ def too_many_inputs(x: List[int], y: List[int], z: Any, z2: Any):
             torch._C._jit_register_shape_compute_graph_for_node(node, too_many_inputs.graph)
 
         self.assertTrue("fewer arguments than schema" in str(error.exception))
+
+    def test_squeeze_dims(self):
+        def foo(input: torch.Tensor):
+            return input.squeeze([0, 2])
+
+        t = torch.jit.script(foo)
+        ten_input = next(t.graph.inputs())
+        ten_input.setType(ten_input.type().with_sizes([1, 2, 1, 4, 3]))
+        torch._C._jit_pass_propagate_shapes_on_graph(t.graph)
+        print(next(t.graph.outputs()).type().symbolic_sizes())
+        self.assertEqual(next(t.graph.outputs()).type().symbolic_sizes(), None)
```

Comment:
doesn't this mean that it's _not_ working?

Reply:
Honestly saying, I don't understand how this works. Also, how to fix them. If you can help in adding/fixing these tests, it would be of great help.

Reply:
Okay, I will check with it.